crates/signs/src/pole.rs

```diff
@@ -124,16 +141,23 @@ struct Pole {
 
 /// Spawn a new pole at this location. There must not be a pole already at this
 /// location.
-struct SpawnPoleEvent(Vec2);
+struct SpawnPoleEvent {
+    pole_location: Vec2,
+    line_start: Vec2,
+}
 
 /// Despawn an existing pole at this location.
 struct DespawnPoleEvent(Vec2);
 
 /// Move an existing pole from one location to another.
-struct MovePoleEvent(Vec2, Vec2);
+struct MovePoleEvent {
+    old_location: Vec2,
+    pole_location: Vec2,
+    line_start: Vec2,
+}
 
 #[derive(Default, Resource)]
-struct Poles(AHashMap<Vec2Ord, Entity>);
+struct Poles(AHashMap<Vec2Ord, [Entity; 2]>);
```

Comment:
This doesn't work correctly. You can reproduce the issue with:

1. open a game & build a second base
2. select the first & second base at once
3. right click on the map to place both poles to a single place

The reason why it doesn't work is pole-merging logic. I.e. when two pole owners place a pole to the same location, only a single pole is created.

IMO the best solution is the creation of a separate module `lines.rs` with the pole-independent line logic there. I would prefer such a solution because decoupling poles & lines would simplify the code, and it would give us more flexibility by allowing poles without lines and vice versa.